woocommerce/google-listings-and-ads
# Get started page UX improvements: add get started with video card

### Changes proposed in this Pull Request:

Part of #1392, this PR add the first card of the new get started page. The first card includes the same marketing video in the [Google Listings & Ads WooCommerce product page](https://woocommerce.com/products/google-listings-and-ads/).

To be able to play the video, I referred to the implementation from 10517-gh-Automattic/woocommerce.com but using [Wistia's JavaScript Player API](https://wistia.com/support/developers/player-api#with-iframe-embeds) to get the video state in order to expand the video height after the user clicks on the video.

To be able to use Wistia's API, I add a custom hook `useScript` and use it to load Wistia's script dynamically in the component.

Note that the video will not be expanded in the mobile version.

Design for desktop: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A120914
Design for mobile: eQ9O4m2flzrcAiDMXkOW0m-fi-7476%3A122828

### Videos:

#### Brave (Chromium)

https://user-images.githubusercontent.com/914406/163765567-62a6b434-8359-4f07-b492-4df607bb9085.mov

### Firefox

https://user-images.githubusercontent.com/914406/163766064-938019f3-b02b-4183-96e8-248e9575ae3c.mov

### Safari

https://user-images.githubusercontent.com/914406/163767314-cdf16c48-8f95-4dd5-a4e6-6c111308d767.mov




### Detailed test instructions:


1. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
2. Check the first card of the get started page, the UI should align with the design.
3. The video will be expanded when playing the video on the desktop.
4. The video won't be expanded when playing the video on the mobile.
5. Click on the `Start listing products` and it will go to the set up page.

### Changelog entry

> Update - Add the first card with a CTA and a video for UX improvements on get started page.
